google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java

```diff
@@ -64,6 +65,20 @@ public void mouseClicked(MouseEvent event) {
             }
           }
         });
+
+    refreshButton.addActionListener(
```

Comment:
Can you add tests for these two new listeners?

Reply:
looking at this, but to have the PR ready to go are there any other comments?